src/api/emulations/nfs.js

```diff
@@ -257,7 +260,13 @@ class NFS {
     }
     return lib.dir_update_file(this.mData.app.connection, this.mData.ref, fileName,
                            file.ref.ref(), version)
-      .then(() => { file.version = version; })  // eslint-disable-line no-param-reassign
+      .then(() => {
+        if (version === CONSTANTS.GET_NEXT_VERSION) {
+          file.version = version + 1; // eslint-disable-line no-param-reassign
```

Comment:
If we're gonna have the rule: `eslint-disable-line no-param-reassign`, let's use it.

@hunterlester, you can avoid linting issues here by just creating a new variable at the top of the relevant block:

`let versionToSet = version // or perhaps 'newVersion' ?`

It'll also read nicer (IMO) for setting the version.